cmd/soroban-rpc/internal/test/get_transactions_test.go

```diff
@@ -0,0 +1,141 @@
+package test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/creachadair/jrpc2"
+	"github.com/creachadair/jrpc2/jhttp"
+	"github.com/stellar/go/keypair"
+	"github.com/stellar/go/toid"
+	"github.com/stellar/go/txnbuild"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/methods"
+)
+
+const (
+	Cursor int64 = 55834578945
+)
+
+// buildTxParams constructs the parameters necessary for creating a transaction from the given account.
+//
+// account - the source account from which the transaction will originate. This account provides the starting sequence number.
+//
+// Returns a fully populated TransactionParams structure.
+func buildTxParams(account txnbuild.SimpleAccount) txnbuild.TransactionParams {
```

Comment:
this seems to build a set options transaction. perhaps this function should be renamed to `buildSetOptionsTxParams()`>